### PR TITLE
Update deployment pool docs

### PR DESCRIPTION
**Pull request by LoekL**

### CODE REVIEW

File: docs/content/configuration-examples/deployment-pool/usage.md [hunk 1]
```diff
@@ -22,10 +22,10 @@ The Deployment pool requires information about the deployments it is to start. T
 
 ## Build your deployment pool image
 
-The Deployment Pool builds from the included Dockerfile to give you a docker image from the code.
+The Deployment Pool builds from the included Dockerfile to give you a docker image from the code. Navigate to the Dockerfile & run:
```
Comment (davej-improbable) [inline docs/content/configuration-examples/deployment-pool/usage.md]: Worth clarifying you need to be in the _parent_ of the `Dockerfile` - i.e. the `docker` folder - not the `deployment-pool` folder itself?

File: docs/content/configuration-examples/deployment-pool/usage.md [hunk 2]
```diff
@@ -61,19 +61,21 @@ On other platforms:
...
 export SPATIAL_REFRESH_TOKEN="[your refresh token]"
+
+# & Use `$SPATIAL_REFRESH_TOKEN` instead of `%SPATIAL_REFRESH_TOKEN%` in the docker command below!
 ```
 
 Once these are in place, you can start the deployment pool using
 
 ```bash
-docker run "gcr.io/[your project id]/deployment-pool" --project "[your spatial project]" --launch-config "[path to your launch config]" --snapshot "[path to your snapshot file]" --minimum-ready-deployments "[number of deployments]"
+docker run -v [local path to launch config]:/launch-config/default_launch.json -v [local path to snapshot file]:/snapshots/default.snapshot -e SPATIAL_REFRESH_TOKEN=%SPATIAL_REFRESH_TOKEN% gcr.io/[your Google project id]/deployment-pool --project "[your SpatialOS project id]" --launch-config "/launch-config/default_launch.json" --snapshot "/snapshots/default.snapshot" --assembly-name "[your uploaded assembly name]" --minimum-ready-deployments 3
```
Comment (davej-improbable) [inline docs/content/configuration-examples/deployment-pool/usage.md]: Should `%SPATIAL_REFRESH_TOKEN%` be `$SPATIAL_REFRESH_TOKEN` here? The comment above says it should be, just unsure why it isn't!

Comment (JoelOtter) [reply docs/content/configuration-examples/deployment-pool/usage.md]: these are Windows instructions. The above comment is about replacing it to work on non-Windows platforms

File: docs/content/configuration-examples/deployment-pool/usage.md (hunk 1 above)
Comment (JoelOtter) [reply docs/content/configuration-examples/deployment-pool/usage.md]: good spot!

Comment (LoekL) [reply docs/content/configuration-examples/deployment-pool/usage.md]: Changed!

File: docs/content/configuration-examples/deployment-pool/usage.md [hunk 3]
```diff
@@ -22,7 +22,7 @@ The Deployment pool requires information about the deployments it is to start. T
 
 ## Build your deployment pool image
 
-The Deployment Pool builds from the included Dockerfile to give you a docker image from the code. Navigate to the Dockerfile & run:
+The Deployment Pool builds from the included Dockerfile to give you a docker image from the code. Navigate to the parent directory of the Dockerfile (`services/docker`) & run:
```
Comment (davej-improbable) [inline docs/content/configuration-examples/deployment-pool/usage.md]: Would suggest using "and" rather than an ampersand to improve readability here.

Comment (LoekL) [reply docs/content/configuration-examples/deployment-pool/usage.md]: Done.

**Pull request merged**